astro/src/styles/colors-and-themes.scss

```diff
@@ -96,11 +96,15 @@ $info:                              #066F80; // branded cyan
 // Secondary Subtle Theme
 $info-text-emphasis:                #004F4F; // cyan-80
 $info-bg-subtle:                    #67DCE0; // cyan-00  
-$info-border-subtle:                #33CCCC; // cyan-10
+$info-border-subtle:                #004F4F; // cyan-80
 
 $info-text-emphasis-dark:           #11BABA; // cyan-20
-$info-bg-subtle-dark:               #003D3D; // cyan-90
-$info-border-subtle-dark:           #006161; // cyan-70
+$info-bg-subtle-dark:               #003D3D; // cyan-90 
+$info-border-subtle-dark:           #11BABA; // cyan-20
```

Comment:
Same here. You're also reversing the dark/light pattern... which is what the subtle border is supposed to be about.